bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/protocol/NikobusCommand.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.nikobus.internal.protocol;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Consumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link NikobusCommand} class holds a command that can be send to Nikobus installation.
+ *
+ * @author Boris Krivonog - Initial contribution
+ */
+public class NikobusCommand {
+    public static class Result {
+        private final Callable<String> callable;
+
+        private Result(String result) {
+            callable = () -> result;
+        }
+
+        private Result(Exception exception) {
+            callable = () -> {
```

Comment:
```suggestion
            callable = () -> throw exception;
```

Reply:
This gives `Syntax error on token(s), misplaced construct(s)`